gui/account/forms.py

```diff
@@ -320,6 +324,16 @@ def clean_bsdusr_home(self):
 
             if home.startswith('/mnt/'):
                 bsdusr_username = self.cleaned_data.get('bsdusr_username', '')
+                volumes = ['/mnt/{}'.format(volume.vol_name) for volume in Volume.objects.all()]
+
+                if self.instance.id:
```

Comment:
Sorry, one more thing I forgot. We probably want to make sure home != user_home, otherwise we will be spending time for no reason.

Reply:
@william-gr ofcourse, and np! It's my mistake I didn't think that through.